cmd/controller-manager/app/options/options.go

```diff
@@ -85,7 +86,8 @@ func NewControllerManagerServer() *ControllerManagerServer {
 // AddFlags adds flags for a ControllerManagerServer to the specified FlagSet.
 func (s *ControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(k8scomponentconfig.IPVar{Val: &s.Address}, "address", "The IP address to serve on (set to 0.0.0.0 for all interfaces)")
-	fs.Int32Var(&s.Port, "port", s.Port, "The port that the controller-manager's http service runs on")
+	fs.Int32Var(&s.Port, "port", 0, "DEPRECATED - use secure-port instead")
```

Comment:
cc @carolynvs isn't there a way to 'dummy' CLI args?

Reply:
@pmorie I don't understand what you want done differently here.  In discussion we agreed to keep the port argument for backward compatibility but to only listen on a secure port.

Reply:
Ahh, @pmorie perhaps you were looking for fs.MarkDeprecated() ?